### PR TITLE
fix(app) dqa software keyboard round3

### PR DESCRIPTION
**Pull request by koji**
# Overview
address software keyboard feedback
change the folder structure for software keyboard components and add `index.css` to each component.

close RAUT-615

# Test Plan
1. run `make -C components dev`
2. go to ODD software keyboard
or
1. push this branch to dev kit/Flex
2. robot settings -> rename robot (CustomKeyboard)
3. robot settings -> wifi settings -> password (NormalKeyboard)

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

File: app/src/atoms/SoftwareKeyboard/CustomKeyboard/__tests__/CustomKeyboard.test.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import { fireEvent } from '@testing-library/react'
 import { renderHook } from '@testing-library/react-hooks'
 import { renderWithProviders } from '@opentrons/components'
-import { CustomKeyboard } from '..'
+import { CustomKeyboard } from '../'
```
Comment (jerader) [inline app/src/atoms/SoftwareKeyboard/CustomKeyboard/__tests__/CustomKeyboard.test.tsx]: what does adding this `/` do?

File: app/src/atoms/SoftwareKeyboard/CustomKeyboard/CustomKeyboard.stories.tsx [hunk 2]
```diff
@@ -5,13 +5,14 @@ import {
   POSITION_ABSOLUTE,
   SPACING,
 } from '@opentrons/components'
-import { InputField } from '../InputField'
-import { CustomKeyboard } from './'
```
Comment (jerader) [inline app/src/atoms/SoftwareKeyboard/CustomKeyboard/CustomKeyboard.stories.tsx:9]: looks like the `/` is here but then you remove it?

**Review by jerader — APPROVED** [review]
lgtm! left a question about the import paths

File: app/src/atoms/SoftwareKeyboard/CustomKeyboard/__tests__/CustomKeyboard.test.tsx (hunk 1 above)
Comment (koji) [reply app/src/atoms/SoftwareKeyboard/CustomKeyboard/__tests__/CustomKeyboard.test.tsx]: oh good catch no needed.
i will remove this.

**Pull request merged**